util/Migrator/SqlServerDbMigrator.cs

```diff
@@ -1,109 +1,21 @@
-﻿using System.Data;
-using System.Reflection;
-using Bit.Core;
-using Bit.Core.Settings;
+﻿using Bit.Core.Settings;
 using Bit.Core.Utilities;
-using DbUp;
-using Microsoft.Data.SqlClient;
-using Microsoft.Extensions.Logging;
 
 namespace Bit.Migrator;
 
 public class SqlServerDbMigrator : IDbMigrator
 {
-    private readonly string _connectionString;
-    private readonly ILogger<SqlServerDbMigrator> _logger;
-    private readonly string _masterConnectionString;
+    private readonly DbMigrator _migrator;
```

Comment:
❓ Everything in this class was duplicated by the core migrator, so now it just uses that like `Setup` and the MSSQL utility. Am I missing anything? Why was this copied?